packages/main/src/components/Toolbar/index.tsx

```diff
@@ -307,10 +307,12 @@ const Toolbar = forwardRef<HTMLDivElement, ToolbarPropTypes>((props, ref) => {
   };
 
   const prevChildren = useRef(flatChildren);
-  const debouncedOverflowChange = useRef(debounce(onOverflowChange, 60));
+  const debouncedOverflowChange = useRef();
 
   useEffect(() => {
-    debouncedOverflowChange.current = debounce(onOverflowChange, 60);
+    if(onOverflowChange){
```

Comment:
This check could be more narrow:

```suggestion
    if (typeof onOverflowChange === 'function' ) {
```